tests/grid/test_constructors.py

```diff
@@ -227,8 +285,12 @@ def test_framed_voronoi_from_dict():
         assert_array_almost_equal(mg.y_of_node[i], true_y[i])
 
     assert_array_almost_equal(
-        mg.adjacent_nodes_at_node[0:2] + mg.adjacent_nodes_at_node[28:30],
-        true_adjacent_nodes_at_node_0_2 + true_adjacent_nodes_at_node_28_30,
+        np.concatenate(
+            (mg.adjacent_nodes_at_node[0:2], mg.adjacent_nodes_at_node[28:30])
+        ),
+        np.concatenate(
+            (true_adjacent_nodes_at_node_0_2, true_adjacent_nodes_at_node_28_30)
+        ),
```

Comment:
@sebastien-lenard It seems like these tests rely on a particular seed. If so, I think they should be modified or removed.

Reply:
Right, I removed those tests